storybook/src/components/Breakout/Breakout.docs.mdx

```diff
@@ -0,0 +1,41 @@
+{/* @license CC0-1.0 */}
+
+import { Canvas, Controls, Markdown, Meta, Primary } from "@storybook/blocks";
+import * as BreakoutStories from "./Breakout.stories.tsx";
+import README from "../../../../packages/css/src/components/breakout/README.md?raw";
+
+<Meta of={BreakoutStories} />
+
+<Markdown>{README}</Markdown>
+
+This example lets an image pop out.
+The text next to it defines the height of the Spotlight.
+On narrower screens, the text moves below the image.
+Ensure that the text is short, or the image tall, to prevent losing the break-out effect.
+
+This component is a copy of `Grid`, so it offers the same props.
+Additionally, `Breakout` allows cells to span multiple columns, and a cell to expand to cover its adjacent gaps and margins.
+
+_The example isn’t very wide on this page.
+And due to how Storybook works, the Grid in the example may incorrectly use 12 columns.
+Both circumstances prevent the image from breaking out if the random example text is long.
```

Comment:
Isn't this a good reason to not use random text here? The example usually doesn't look the way we want it to, which is a bit of a shame.

Reply:
Yes, it is.